### PR TITLE
azure standard install fix - configure DNS astronomer/issues/#1971

### PR DESCRIPTION
**Pull request by sunkickr**
Removed and added information about configuring DNS during standard Installation to update doc see https://github.com/astronomer/issues/issues/1971

### CODE REVIEW

**Review by paolaperaza — APPROVED** [review]
@sunkickr I can't verify the technical accuracy of these changes but copy and formatting wise they look good 😊 

@ianstanton Feel free to review + merge as you see fit!

File: enterprise/next/03_install/03_azure/01_install-azure-standard.md [hunk 1]
```diff
@@ -236,7 +218,7 @@ global:
 #################################
 nginx:
   # IP address the nginx ingress should bind to
-  loadBalancerIP: YOUR_EXTERNAL_IP_HERE
+  loadBalancerIP:
```
Comment (ianstanton) [inline enterprise/next/03_install/03_azure/01_install-azure-standard.md:221]:
```suggestion
  loadBalancerIP: ~
```

File: enterprise/next/03_install/03_azure/01_install-azure-standard.md [hunk 2]
```diff
@@ -349,14 +331,61 @@ astronomer-registry-0                                      1/1     Running
 
 If you are seeing issues here, check out our [guide on debugging your installation](/docs/enterprise/v0.16/troubleshoot/debug-install/).
 
+## 9. Configure DNS
+
+Now that you've successfully installed Astronomer, a new Load Balancer will have spun up in your Azure account. This Load Balancer routes incoming traffic to our NGINX ingress controller.
+
+Run `kubectl get svc -n astronomer` to view your Load Balencer's External IP Address, located under the `EXTERNAL-IP` column for the `astronomer-nginx` service.
```
Comment (ianstanton) [inline enterprise/next/03_install/03_azure/01_install-azure-standard.md:338]:
```suggestion
Run `kubectl get svc -n astronomer` to view your Load Balancer's External IP Address, located under the `EXTERNAL-IP` column for the `astronomer-nginx` service.
```

File: enterprise/v0.16/03_install/03_azure/01_install-azure-standard.md [hunk 3]
```diff
@@ -236,7 +218,7 @@ global:
 #################################
 nginx:
   # IP address the nginx ingress should bind to
-  loadBalancerIP: YOUR_EXTERNAL_IP_HERE
+  loadBalancerIP:
```
Comment (ianstanton) [inline enterprise/v0.16/03_install/03_azure/01_install-azure-standard.md:221]:
```suggestion
  loadBalancerIP: ~
```

File: enterprise/v0.16/03_install/03_azure/01_install-azure-standard.md [hunk 4]
```diff
@@ -349,14 +331,61 @@ astronomer-registry-0                                      1/1     Running
 
 If you are seeing issues here, check out our [guide on debugging your installation](/docs/enterprise/v0.16/troubleshoot/debug-install/).
 
+## 9. Configure DNS
+
+Now that you've successfully installed Astronomer, a new Load Balancer will have spun up in your Azure account. This Load Balancer routes incoming traffic to our NGINX ingress controller.
+
+Run `kubectl get svc -n astronomer` to view your Load Balencer's External IP Address, located under the `EXTERNAL-IP` column for the `astronomer-nginx` service.
```
Comment (ianstanton) [inline enterprise/v0.16/03_install/03_azure/01_install-azure-standard.md:338]:
```suggestion
Run `kubectl get svc -n astronomer` to view your Load Balancer's External IP Address, located under the `EXTERNAL-IP` column for the `astronomer-nginx` service.
```

File: enterprise/next/03_install/03_azure/01_install-azure-standard.md [hunk 5]
```diff
@@ -205,7 +187,7 @@ As a next step, create a file named `config.yaml` in an empty directory.
 
 For context, this `config.yaml` file will assume a set of default values for our platform that specify everything from user role definitions to the Airflow images you want to support. As you grow with Astronomer and want to customize the platform to better suit your team and use case, your `config.yaml` file is the best place to do so.
 
-In the newly created file, copy the example below and replace `baseDomain` and `smtpUrl` with your own values. For more example configuration files, go [here](https://github.com/astronomer/astronomer/tree/release-0.14/configs).
+In the newly created file, copy the example below and replace `baseDomain` and `smtpUrl` with your own values. If you would like to use your own external IP address, add the address to `loadBalancerIP`. If you want Azure to make one for you, leave this section blank. For more example configuration files, go [here](https://github.com/astronomer/astronomer/tree/release-0.14/configs).
```
Comment (ianstanton) [inline enterprise/next/03_install/03_azure/01_install-azure-standard.md:190]:
```suggestion
In the newly created file, copy the example below and replace `baseDomain` and `smtpUrl` with your own values. If you would like to use your own external IP address, add the address to `loadBalancerIP`. Azure will dynamically generate an IP address if this value is left blank. For more example configuration files, go [here](https://github.com/astronomer/astronomer/tree/release-0.14/configs).
```

File: enterprise/v0.16/03_install/03_azure/01_install-azure-standard.md [hunk 6]
```diff
@@ -205,7 +187,7 @@ As a next step, create a file named `config.yaml` in an empty directory.
 
 For context, this `config.yaml` file will assume a set of default values for our platform that specify everything from user role definitions to the Airflow images you want to support. As you grow with Astronomer and want to customize the platform to better suit your team and use case, your `config.yaml` file is the best place to do so.
 
-In the newly created file, copy the example below and replace `baseDomain` and `smtpUrl` with your own values. For more example configuration files, go [here](https://github.com/astronomer/astronomer/tree/release-0.14/configs).
+In the newly created file, copy the example below and replace `baseDomain` and `smtpUrl` with your own values. If you would like to use your own external IP address, add the address to `loadBalancerIP`. If you want Azure to make one for you, leave this section blank. For more example configuration files, go [here](https://github.com/astronomer/astronomer/tree/release-0.14/configs).
```
Comment (ianstanton) [inline enterprise/v0.16/03_install/03_azure/01_install-azure-standard.md:190]:
```suggestion
In the newly created file, copy the example below and replace `baseDomain` and `smtpUrl` with your own values. If you would like to use your own external IP address, add the address to `loadBalancerIP`. An IP Address will be dynamically generated in Azure if this value is left blank. For more example configuration files, go [here](https://github.com/astronomer/astronomer/tree/release-0.14/configs).
```

**Comment by jwitz** [comment]
Thanks for these edits @ianstanton ! I went ahead and committed them so that we can move forward with this PR. Everything else looks good to me as well, so I'm gonna merge.

**Pull request merged**